packages/core/src/factory.ts

```diff
@@ -37,23 +39,41 @@ export function factory<Resource, Input, Output, Config, LoadOpts>(
         // build the config for this request
         const configMerger = configMergerFactory(defaultConfig, customConfig, c);
         const configObj: Config | undefined = configMerger(input);
```

Comment:
This is probably an old leftover.

```suggestion
        const configObj: Config = configMerger(input);
```